content/styles/main.less

```diff
@@ -5,7 +5,7 @@
 body {
   margin: 0 auto;
   padding: 20px;
-  max-width: 800px - 2*20px;
+  max-width: 900px - 2*20px;
```

Comment:
@rtaelman: Why? (Menu width?)

Reply:
Okay, let's fix the menu at some later stage, by making it hamburgerable or something.